src/microbe_stage/MicrobeSpecies.cs

```diff
@@ -121,4 +122,47 @@ public override object Clone()
 
         return result;
     }
+
+    public float BaseOsmoregulationCost()
+    {
+        return Organelles.Sum(organelle => organelle.Definition.HexCount)
+            * MembraneType.OsmoregulationFactor
+            * Constants.ATP_COST_FOR_OSMOREGULATION;
+    }
+
+    public float BaseSpeed()
```

Comment:
I think this code should be merged with the one in the editor calculating a species speed so we don't have this quite complex piece of code twice.